README.md

```diff
@@ -1 +1,47 @@
-# mobile-portfolio-setup
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# MY Portfolio
+
+
+## Built With
+
+- HTML and CSS
+- Webhint
+- Styelint
+
+<!-- ## Live Demo (if available)q
+
+[Live Demo Link](https://livedemo.com) -->
+
+
+<!-- ## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.** -->
+
+## Authors
+
+👤 **DERRICK**
+
+- GitHub: [@githubhandle](https://github.com/derrick1451)
+- Twitter: [@twitterhandle](https://twitter.com/derrickayle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/okongoderrick)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+
```

Comment:
- [ ] [OPTIONAL] Please kindly add the MIT license file and link to it as in the microverse [template](https://github.com/microverseinc/readme-template)